scripts/install

```diff
@@ -87,6 +87,17 @@ ynh_script_progression --message="Configuring $app..." --weight=8
 # Set the mysql.utf8mb4 config to true in config.php
 exec_occ config:system:set mysql.utf8mb4 --type boolean --value="true"
 
+# Set default language in config.php
+raw_language=$(echo "$language" | awk -F'_' '{print $1}')
+exec_occ config:system:set default_language --value="$raw_language"
```

Comment:
shouldn't it be just $language here ?

Reply:
Nope, Nextcloud's `default_language` uses ISO_639-1 language codes such as `en` for English, see https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#default-language

That's why we need to cut the first part from `$language`, e.g. take `en` from `en_US`.